spec/admin/target_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe 'Admin Targets', type: :request do
+  let(:admin_user) { FactoryBot.create(:admin_user) }
+
+  before do
+    sign_in admin_user
+  end
+
+  describe 'Show all targets' do
```

Comment:
minor comment maybe it's not the best description because this is an index method.

Reply:
Done! I add the screenshot to the description of the PR.